keyboards/primekb/prime_e/keymaps/peott-fr/keymap.c

```diff
@@ -0,0 +1,103 @@
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+enum custom_layers {
+  _QWERTY,
+  _LEFTHAND,
+  _NUM,
+  _FUNC
+};
+
+#define SPC_LFT LT(_LEFTHAND, KC_SPC)
+#define FN_NUM LT(_NUM, KC_MNXT)
+#define BSP_FUNC LT(_FUNC, KC_BSPC)
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+	[_QWERTY] = LAYOUT(
+			KC_ESC, KC_Q, KC_W, KC_E, KC_R, KC_T, 	      KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, KC_RBRC, 
+              		LCTL_T(KC_TAB), KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT, 
+	      		KC_LSPO, KC_Z, KC_X, KC_C, KC_V, KC_B, 	      KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_RSPC, 
+	      		LCTL_T(KC_MPRV), LALT_T(KC_MPLY), FN_NUM,     SPC_LFT, BSP_FUNC, LGUI_T(KC_DEL), RALT_T(KC_HOME), RCTL_T(KC_END)
+        ),
+	[_LEFTHAND] = LAYOUT(
+			KC_TRNS, KC_TRNS, KC_TRNS, KC_UP, KC_TRNS, KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, 
+			KC_TRNS, KC_TRNS, KC_LEFT, KC_DOWN, KC_RGHT, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_ENT, 
+			KC_TRNS, KC_CALC, KC_MYCM, KC_TRNS, KC_ENT, KC_BSPC,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, 
+			KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS, KC_TRNS, KC_PGDN, KC_PGUP
+	),
+	[_NUM] = LAYOUT(
+			KC_GRV, KC_1, KC_2, KC_3, KC_4, KC_5,                 KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_EQL, 
+			KC_CAPS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_BSLS, 
+			KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, 
+			KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+	),
+	[_FUNC] = LAYOUT(
+			RESET, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5,             KC_F6, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, 
+			KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_PSCR, KC_SLCK, KC_PAUS, KC_INS, KC_TRNS, KC_TRNS, 
+			KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, RGB_TOG, RGB_MOD, RGB_HUI, RGB_SAI, RGB_VAI, RGB_SPI, KC_TRNS, 
+			KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+	)
+};
+
+void matrix_init_user(void) {
+  // set CapsLock LED to output and low
+  setPinOutput(B1);
+  writePinLow(B1);
+  // set NumLock LED to output and low
+  setPinOutput(B2);
+  writePinLow(B2);
+  // set ScrollLock LED to output and low
+  setPinOutput(B3);
+  writePinLow(B3);
+}
+
+void matrix_scan_user(void) {
+
+}
+
+void led_set_user(uint8_t usb_led) {
```

Comment:
Use https://docs.qmk.fm/#/feature_led_indicators if possible (I don't know if this board has been converted).

Reply:
Hi! Unfortunately I don't understand the raw code/functions. My more complex stuff is mostly copy/paste.
The keymap functions as desired so I don't think there's anything wrong/incorrect with the code.

Is this blocking the merge?